css/portal-dashboard/all-responses-popup/student-responses-popup.less

```diff
@@ -0,0 +1,14 @@
+@import "../variables";
+
+.popup {
+    position: fixed;
+    bottom: 0;
+    padding: 0;
+    box-sizing: border-box;
+    box-shadow: -1px 1px 4px 0px rgba(0, 0, 0, 0.35);
+    background-color: #ffffff;
+    transition: all 0.5s ease;
+    width: 100%;
+    height: 100%;
+    font-weight: bold;
+}
```

Comment:
You might add in `z-index: 1;` to bring this on top of other content.  You'll eventually need to place this over everything else, and I think it makes things make more sense visually by adding it now.